typescript/sdk/src/deploy/AbacusDeployer.ts

```diff
@@ -67,8 +67,11 @@ export abstract class AbacusDeployer<
       },
     );
     const configChains = Object.keys(this.configMap) as Chain[];
-    this.logger(`Start deploy to ${configChains}`);
-    for (const chain of configChains) {
+    const targetChains = this.multiProvider
+      .chains()
+      .filter((chain) => configChains.includes(chain));
+    this.logger(`Start deploy to ${targetChains}`);
+    for (const chain of targetChains) {
```

Comment:
Shouldn't we skip chains to remove here?

Reply:
remove is on `CoreConfig`
`Config` here can be `any` if you remember 😈

Reply:
skipped in coredeployer